tests/providerTest.js

```diff
@@ -403,6 +403,29 @@ describe('provider testing', () => {
           chai.assert.fail('Failed to assign last created tag to last created file')
         }
       }
+    },
+    'a file is uploaded using TUS protocol': (setup, parameters) => {
```

Comment:
you only the creation of a TUS file, not an actual upload 
CC @swoichha

Reply:
@individual-it @swoichha 
The tests are like `returns TUS support information when TUS headers are set for a list call`
Please, guide me what needs to be done as given for such tests.